<launch>
    <rosparam command="load" file="$(find livox_camera_calib)/config/multi_calib.yaml" />
    
    <node pkg="livox_camera_calib" type="calib_tune" name="calib_tune" output="screen"/>

    <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure"/>

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find livox_camera_calib)/rviz_cfg/calib_tune.rviz"/>

    <!-- <node pkg="tf" type="static_transform_publisher" name="livox_bc" args="0 0 0 0 0 0 1 world livox 100"/> -->

</launch>
